src/components/ModalTwo.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
There is a lot of repetition here, delete this modal component and let's use instead just the modal above

Adjust that modal to be able to handle all the cases you need